src/Uno.UI.Runtime.Skia.Win32/Win32Host.cs

```diff
@@ -73,6 +76,16 @@ static Win32Host()
 		ApiExtensibility.Register<DragDropManager>(typeof(IDragDropExtension), manager => new Win32DragDropExtension(manager));
 		ApiExtensibility.Register<ContentPresenter>(typeof(ContentPresenter.INativeElementHostingExtension), o => new Win32NativeElementHostingExtension(o));
 		ApiExtensibility.Register<CoreWebView2>(typeof(INativeWebViewProvider), o => new Win32NativeWebViewProvider(o));
+
+		if (Type.GetType("Uno.UI.MediaPlayer.Skia.Win32.Win32MediaPlayerPresenterExtension, Uno.UI.MediaPlayer.Skia.Win32") is { } mediaPresenterExtensionType)
```

Comment:
Let's explain why we need this, and also when we'll be able to remove it (once we remove Wpf support)